config/schedule.yml

```
@@ -0,0 +1,6 @@
+cancel_reservations_job:
+  cron: "*/5* * * *"
```

```suggestion
  cron: "*/5 * * * *"
```
That missing space is preventing this cronjob from being scheduled :)